giskard/core/suite.py

```diff
@@ -66,12 +66,12 @@ def parse_function_arguments(client, project_key, function_inputs):
             arguments[value["name"]] = str(value["value"])
         elif value["type"] == "bool":
             arguments[value["name"]] = bool(value["value"])
-        elif value["type"] == "kwargs":
+        elif value["type"] == "Kwargs":
```

Comment:
Is this a typo now or was it a typo before?

Reply:
It was before, `Kwargs` are uppercase in the Hub.